src/fal/logger.py

```diff
@@ -0,0 +1,119 @@
+from dataclasses import dataclass
+
+from dbt.logger import log_manager
+from dbt.events.functions import fire_event
+from dbt.events.base_types import (
+    TestLevel as _TestLevel,
+    DebugLevel as _DebugLevel,
+    InfoLevel as _InfoLevel,
+    WarnLevel as _WarnLevel,
+    ErrorLevel as _ErrorLevel,
+)
+
+
+class LOGGER:
```

Comment:
ATL: why is the logger is all capitalized?

Reply:
because we never instantiate it. Maybe this is not the correct pattern to follow and we should instead offer the functions directly?

Then importing and using like:

```
import fal.logger as logger

logger.info(...)
```

Reply:
Both seems to work for me, but if we are using it just for the namespace-ing I'd be also fine with a single instance.